openapi_test.go

```diff
@@ -303,7 +303,7 @@ func TestServer_OutputOpenApiSpec(t *testing.T) {
 		})
 
 		document := s.OutputOpenAPISpec()
-		require.Len(t, document.Paths.Map(), 1)
+		require.Len(t, document.Paths.Map(), 0)
```

Comment:
Require Len 0 is about using Empty


```suggestion
		require.Empty(t, document.Paths.Map())
```